code-examples/quarkus/kafka-quickstart/bindings/kafka/bootstrapServers

```diff
@@ -0,0 +1 @@
+youcan-use--r-n-oyykfggso-rge-mwi-uoro.kafka.devshift.org:443/bindings/rhoas-kafka-d-youcan-use-your-cluster-akc
```

Comment:
This looks like a file that got committed by mistake.

Reply:
Those are examples placeholders to show users how binding works. Typically we should not commit them but for purpose of example we want to show desired structure

Reply:
so how does these files get used?

Reply:
service binding allows developers use it in the development when specifying SERVICE_BINDING_ROOT property in application.properties. My guide documents how to use it from development to production (As opposed to existing guide that is purely dev focused)